Source/WebCore/rendering/RenderElement.cpp

```diff
@@ -1340,7 +1340,7 @@ bool RenderElement::repaintAfterLayoutIfNeeded(const RenderLayerModelObject* rep
     }();
 
     if (!repaintContainer)
-        repaintContainer = &view();
+        repaintContainer = { view() };
```

Comment:
I doubt you need the curly brackets here.

Reply:
The braces were removed.